modules/ROOT/pages/whatsnew.adoc

```diff
@@ -20,19 +20,22 @@ include::partial$_show_page_header_block.adoc[]
 --
 
 Do not deploy Eventing/Sync Gateway until all SGW nodes are at version 3.2.
-For older Sync Gateway versions that do not write import XATTRs, Eventing functions may encounter infinite recursions and duplicate mutations if deployed in a mixed mode SGW environment.
-This can only happen during a mixed mode of SGW with a 3.2 version and an older version during an upgrade and a new Eventing/Sync Gateway function is deployed.
+For earlier Sync Gateway versions that do not write import XATTRs, Eventing functions may encounter infinite recursions and duplicate mutations if deployed in a mixed mode SGW environment.
+This can only happen during a mixed mode of SGW with a 3.2 version and an earlier version during an upgrade and you deploy a new Eventing/Sync Gateway function.
```

Comment:
Not sure about this paragraph. 🤔

Could you try rewording it?

"This can only happen in a mixed installation consisting of SGW 3.2 and an earlier version when you deploy a new Eventing function during an upgrade."